src/databricks/labs/ucx/assessment/assessment.py

```diff
@@ -60,20 +68,138 @@ def _verify_ws_client(w: WorkspaceClient):
             msg = "Current user is not a workspace admin"
             raise RuntimeError(msg)
 
-    def table_inventory(self, executor: CommandExecutor | None = None):
-        logger.info("Started dataset inventorization...")
-        ce = self._get_command_executor(executor, language=Language.SCALA)
-        params = {"SCHEMA": self._inventory_schema}
-        cmd_code = self._get_command("create_table_inventory.scala", params=params)
-        command_output = ce.run(cmd_code)
-        logger.debug(command_output)
-        if executor is None:
-            self._remove_command_executor()
-        logger.info("Completed dataset inventorization...")
-
-    def compile_report(self):
-        logger.info("Started report compilation...")
-        ce = self._get_command_executor(None, language=Language.SCALA)
-        self.table_inventory(ce)
-        self._remove_command_executor()
-        logger.info("Completed report compilation...")
+    def generate_ext_loc_list(self):
```

Comment:
Why do we need these methods? Can't we just call these two lines from runtime.py?.. the assessment class seems mostly redundant

Reply:
It is a matter of style. I can remove it.

Reply:
Please remove :)